test-lib.sh

```diff
@@ -16,6 +16,8 @@
 
 # may be redefined in the specific container testfile
 EXPECTED_EXIT_CODE=0
+UNSTABLE_TESTS=""
```

Comment:
What happens when I have `declare -a UNSTABLE_TESTS=(pipenv-test-app)` in my `test/run`. Does this line redefines the variable?

Reply:
In this order it would work:
```
$ UNSTABLE_TESTS=""
$ declare -a UNSTABLE_TESTS=(pipenv-test-app)
$ echo $UNSTABLE_TESTS
pipenv-test-app
```
However, the `testlib.sh` must be included in the script before declaring the array and that is inconvenient. I have changed the implementation a little, thanks.